schemas/telemetry/_common-schema.json

```diff
@@ -100,6 +100,48 @@
         "type": "string"
       },
       "readOnly": true
+    },
+    "telemetry": {
+      "type": "object",
+      "additionalProperties": true,
```

Comment:
question/ Why also adding `"additionalProperties": true`? It looks unrelated to `device` and `os` change.

Reply:
because the error-schema and debug-schema also add properties to the telemetry object, so this is to enable a merge